From 9e37fa6192e27cee7f59f17d879a38610abf2803 Mon Sep 17 00:00:00 2001
From: Andre <andre.netzeband@hm.edu>
Date: Fri, 5 May 2017 23:07:16 +0200
Subject: [PATCH 3/5] Pass configuration type to subprojects.

---
 packaging/3rdParty-devel/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/3rdParty-devel/CMakeLists.txt b/packaging/3rdParty-devel/CMakeLists.txt
index 7c1577c..5612a16 100644
--- a/packaging/3rdParty-devel/CMakeLists.txt
+++ b/packaging/3rdParty-devel/CMakeLists.txt
@@ -59,7 +59,7 @@ message(STATUS "CMAKE_SOURCE_DIR = ${CMAKE_SOURCE_DIR}")
 
 set(EXTERNALS_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/externals" CACHE PATH "")
 set(EXTERNALS_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/3rdParty" CACHE PATH "")
-set(EXTERNALS_CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALS_INSTALL_DIR} CACHE PATH "")
+set(EXTERNALS_CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALS_INSTALL_DIR} -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE})
 
 set_property(DIRECTORY PROPERTY EP_BASE ${EXTERNALS_PREFIX})
 
-- 
2.11.1.windows.1

