From 225fbae916296566adb76edc6a279534e458068f Mon Sep 17 00:00:00 2001
From: Andre <andre.netzeband@hm.edu>
Date: Fri, 5 May 2017 23:07:52 +0200
Subject: [PATCH 4/5] Added install target for CLion.

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3842f38..1a89aab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,3 +148,7 @@ ADD_CUSTOM_TARGET(${_UNINST_TGT_NAME} "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BIN
 # Configure CPack.
 INCLUDE(cmake/packagemaker.cmake)
 
+add_custom_target(install_speed-dreams-2
+		$(MAKE) install
+		DEPENDS speed-dreams-2
+		COMMENT "Installing speed-dreams-2")
\ No newline at end of file
-- 
2.11.1.windows.1

